Backend/src/main/java/com/developer/auctionapp/entity/Role.java

```diff
@@ -34,18 +35,18 @@ public class Role {
     public Role(final Long id, final String name) {
         this.id = id;
         this.name = name;
-        users = null;
+        users = new ArrayList<>();
     }
 
     public Role(final String name) {
         this.name = name;
-        users = null;
+        users = new ArrayList<>();
     }
 
     public Role() {
-        id = null;
-        name = null;
-        users = null;
+        id = 0l;
+        name = "";
```

Comment:
it doesn't make sense, if you have a constructor that sets your values to an empty string or 0, then you don't need to have a final variable, we talked about that